src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordPrintMembers.cs

```diff
@@ -48,21 +48,14 @@ protected override DeclarationModifiers MakeDeclarationModifiers(DeclarationModi
 #endif
             return result;
 
-            MethodSymbol? virtualPrintInBase()
+#if DEBUG
+            bool modifiersAreValid(DeclarationModifiers modifiers)
```

Comment:
What happens when the surrounding `#if DEBUG` directives are removed from here and the call site?

Reply:
We'd emit non-product code into the shipping product.

---
In reply to: [603564256](https://github.com/dotnet/roslyn/pull/52012#discussion_r603564256) [](ancestors = 603564256)